docs/reference/modules/express.md

```diff
@@ -0,0 +1,84 @@
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+|`address` | String | Apostrophe listens for connections on all interfaces (`0.0.0.0`) unless this option is set to another address. If the `ADDRESS` environment variable is set, it is used instead. |
+|`apiKeys` | Object | Configure API keys for request authentication. See [the authentication guide](/reference/api/authentication.md#api-keys) for more. |
+|`bodyParser` | Object | The `json` and `urlencoded` properties of this object are merged with Apostrophe's default options to be passed to the [`body-parser` npm module's](https://www.npmjs.com/package/body-parser) `json` and `urlencoded` methods. |
+|`csrf` | Boolean/Object | Set to `false` to disable CSRF protection or to an object with `name` property to customize the CSRF cookie name. See below. |
+|`expressBearerToken` | Object | An options object passed to [`express-bearer-token`](https://www.npmjs.com/package/express-bearer-token) for the bearer token middleware. |
+|`port` | Integer | Apostrophe listens for connections on port `3000` unless this option is set to another port. If the `PORT` environment variable is set, it is used instead. |
+|`session` | Object | Properties of the `session` option are used to create the session middleware. See below. |
+|`trustProxy` | Boolean | Enables the [trust proxy option for Express](https://expressjs.com/en/api.html#trust.proxy.options.table). Set to `true` to tell the Express app to  respect `X-Forwarded-* ` headers. This is helpful when Apostrophe is generating `http:` URLs even though a proxy like nginx is being used to serve it over `https:`. |
+
+### `session`
+
+The `session` options object is passed to the
+[express-session](https://npmjs.org/package/express-session) function. If each is not otherwise specified, Apostrophe enables the following defaults:
+
+```javascript
+{
+  // Does not save sessions until something is stored in them. This greatly
+  // reduces `aposSessions` collection size.
+  saveUninitialized: false,
+  // We are using the 3.x mongo store which is compatible with the
+  // `resave: false` option, preventing the vast majority of session-related
+  // race conditions.
+  resave: false,
+  // Always update the cookie, so that each successive access revives your
+  // login session timeout.
+  rolling: true,
+  // This option should be customized in every project.
+  secret: 'you should have a secret',
+  name: self.apos.shortName + '.sid',
+  cookie: {}
+}
+```
+
+If you want to use another session store, you can pass an instance, but it's easier to let Apostrophe do the work of setting it up:
```

Comment:
By default Apostrophe stores sessions in MongoDB so it is not necessary to install another solution. If you want to use another session store, you can pass an instance as the `store` sub-option, but it's easier to let Apostrophe do the work of setting it up: